cirq-core/cirq/experiments/xeb_fitting.py

```diff
@@ -136,6 +137,38 @@ def get_initial_simplex_and_names(
         """Return an initial Nelder-Mead simplex and the names for each parameter."""
 
 
+def phased_fsim_angles_from_gate(gate: 'cirq.Gate') -> Dict[str, float]:
+    """For a given gate, return a dictionary mapping '{angle}_default' to its noiseless value
+    for the five PhasedFSim angles."""
+    defaults = {
+        'theta_default': 0.0,
+        'zeta_default': 0.0,
+        'chi_default': 0.0,
+        'gamma_default': 0.0,
+        'phi_default': 0.0,
+    }
+    if gate == SQRT_ISWAP:
+        defaults['theta_default'] = -np.pi / 4
```

Comment:
I think we should set also phi_default to np.pi / 24 here and below. This should improve the optimization speed? I'll try this setting as soon I have a chance. We can change this in a follow-up?

Reply:
I think this would be a dangerous semantic to introduce in `cirq-core`, which is where this function currently lives. Please be aware that if you set `characterize_phi=False`, the fixed value will be set to `phi_default`. I don't know if this would help or hurt when you're only optimizing phases.

Options:
1) [my preference] have a wrapper like `SqrtISwapXEBOptions` (`GoogleSqrtISwapXEBOptions`) that sets {angle}_default values and tell users to use it instead of the automatically-determine-from-gate when you're using SqrtIswap on a google processor because it can help with phi characterization
2) Inject a different `phased_fsim_angles_from_gate` if you're running via `cirq_google.workflow`
3) [radical] Introduce a new sqrt-iswap-like gate defined to ideally have phi of np.pi/24 and have people write their algorithms with this gate. Then the angle determination would work automatically. You could still treat it as an iswap gate but you'd have to explicitly neglect the small conditional phase

Reply:
I like option 2 but maybe let's play with this future a little bit if that would make any difference at all?

Reply:
I included the gate-angle-inference function as an argument to the method to support option 2